test/console_TEST.cc

```diff
@@ -83,6 +85,15 @@ TEST(ConsoleTest, MacroExpansionTest_ItShouldCompile)
   }
 }
 
+//////////////////////////////////////////////////
+TEST(ConsoleTest, StdoutStderrOutput)
+{
+  CONSOLE_BRIDGE_logDebug("Testing Log");
+  CONSOLE_BRIDGE_logInform("Testing Log");
+  CONSOLE_BRIDGE_logWarn("Testing Log");
+  CONSOLE_BRIDGE_logError("Testing Log");
```

Comment:
Other than coverage, this isn't a super helpful test as there are no `ASSERT`/`EXPECT` here.  Most of the point of the `OutputHandlerString` class is so we can capture these and test against them.  So with that in mind, I'd suggest just reusing the `OutputHandlerString` class here, and calling it at the various levels.

Alternatively, if you want to test the default handler for stdout, you can do something like https://github.com/ros/console_bridge/pull/74.

Reply:
The test is just to add the default handler code path to the test file. I don't have a good solution to capturing stdout/stderr for testing purposes. I just ran into this issue with https://github.com/ros2/rcutils/pull/258. It looks like it is currently not recommended to use `testing::internal::CaptureStdout()` since it's in an internal namespace.

I've added expect_no_throw statements for now. I'm not yet convinced it's actually adding coverage. I'll remove the test if it doesn't actually affect coverage.

Reply:
Looks like this test adds a couple of lines and bumps coverage over 95% with `setLogLevel` to debug.